common/utils/loadUserData.js

```diff
@@ -108,8 +108,9 @@ export async function loadUserData(api, store, alerts) {
       USER_QUERY,
       {
         id: userId,
-        activityAfter: now() - DAY * DEFAULT_NB_DAYS_MISSIONS_HISTORY,
-        nbFirstMissions: DEFAULT_NB_FIRST_MISSIONS
```

Comment:
`nbFirstMissions` wasn't used, the back was (and still is) limiting to 200 the returned data independently to this variable